.github/workflows/main.yml

```diff
@@ -94,12 +94,6 @@ jobs:
           key: bazel-${{ hashFiles('**/.gitmodules') }}
           restore-keys: bazel-
 
-      - name: Setup bazel
```

Comment:
Bazelisk is now included in the github-runner image.
Just need to make sure that we're using a Bazel version that works with the conformance tests.

Reply:
Note: looks like bazel introduced a _behavioral_ change in v7, that lets the conformance tests build fail.